ui/scss/component/_button.scss

```diff
@@ -84,4 +84,10 @@ $button-focus-shift: 12%;
 .button--flat
 {
   box-shadow: none !important;
-}
\ No newline at end of file
+}
+
+.button-submit {
```

Comment:
Why do we need this? And it should probably be `button--submit` if this is a specific state or type of a button.

Reply:
the current styles are for `<a></a>` and don't work with an actual `<button></button>` ^^